lib/auth/grpcserver.go

```diff
@@ -824,6 +840,9 @@ func (g *GRPCServer) GetCurrentUserRoles(_ *emptypb.Empty, stream authpb.AuthSer
 	return nil
 }
 
+// GetUsers returns all users.
+// TODO(tross): DELETE IN 16.0.0
+// Deprecated: use [usersv1.Service.ListUser] instead.
```

Comment:
nit/FYI: if we were to follow https://cloud.google.com/apis/design/standard_methods#list the method name would be ListUsers. There are some small differences in RPC signatures too.